prep-javascript-variables/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>JavaScript Variables</title>
+  </head>
+  <body>
+    <script>
+      const FullName = 'Lucas Yasumura';
```

Comment:
In JavaScript, variable names should always be camelCased. So this should be `fullName` instead.

The only time you capitalize a name in JavaScript is when you are creating a `class` or a Component in React (covered in the full-immersion program)